src/robot/output/logger.py

```diff
@@ -202,7 +202,7 @@ def _log_message(self, msg, no_cache=False):
             and self._output_file.is_logged(msg)
         ):
             self._log_message_parents[-1].body.append(msg)
-        if msg.level in ("WARN", "ERROR"):
+        if msg.console or msg.level in ("WARN", "ERROR"):
```

Comment:
This looks odd. Shouldn't it be enough to use `msg.console` also here? Could it be that this was needed when `librarylogger.write` still had `also_console=False`? I can test that locally after merging the PR.